requirments.txt

```diff
@@ -7,3 +7,4 @@ pyasn1>=0.1,<0.2
 markupsafe>=0.23,<1.0
 click>=5.0,<7.0
 gitpython>=1.0,<1.1
+python-ldap
```

Comment:
We should pin this to the current version. It looks like 2.4.27 is the current version, so maybe set this to `python-ldap>=2.4,<2.5`.